addons/website_sale/static/src/js/sale_variant_mixin.js

```diff
@@ -640,17 +640,17 @@ var VariantMixin = {
      * information about the last selected combination is useful. All
      * intermediary rpc can be ignored and are therefore best not done at all.
      *
-     * The DropMisordered is to make sure slower rpc are ignored if the result
+     * The KeepLast is to make sure slower rpc are ignored if the result
```

Comment:
The keepLast is to make sure we only consider the result of the last call, when several (asynchronous) calls are done in parallel.